plugins/videobasedtracker/HDKLedIdentifierFactory.cpp

```diff
@@ -169,6 +169,16 @@ namespace vbtracker {
         return ret;
     }
 
+    LedIdentifierPtr createHDKUnifiedLedIdentifier() {
+        LedIdentifierPtr ret;
+        std::vector<std::string> patterns = OsvrHdkLedIdentifier_SENSOR0_PATTERNS;
+        patterns.insert(end(patterns),
```

Comment:
Where is `begin()` and `end()` defined? Did you pull `std::begin()` into the current namespace? Some ADL magic?